litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/TuneWorkflowSteps/TargetApplication.tsx

```diff
@@ -84,136 +178,365 @@ const TargetApplication: React.FC<TargetApplicationProp> = ({
+        version: 'v1alpha1',
+        resource: 'rollouts',
+      });
+    } else {
+      setGVRObj({
+        group: '',
+        version: '',
+        resource: '',
+      });
+    }
+  }, []);
+
   return (
     <div>
-      <Typography>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua.
+      <Typography className={classes.annotationInfo}>
+        {t('createWorkflow.tuneWorkflow.verticalStepper.annotationInfo')}
       </Typography>
       <br />
       <div className={classes.inputDiv}>
-        <InputField
-          label="Target Application"
-          value={targetApp.target}
-          onChange={(event) => {
-            setTargetApp({ ...targetApp, target: event.target.value });
-          }}
-        />
-        <br />
-        {isCustom && (
-          <>
-            <InputField
-              label="Pod"
-              value={targetApp.pod}
-              onChange={(event) => {
-                setTargetApp({ ...targetApp, pod: event.target.value });
+        {/* Annotation Check  */}
+        <div style={{ display: 'flex' }}>
+          <Typography className={classes.annotation}>
+            {t('createWorkflow.tuneWorkflow.verticalStepper.annotation')}
+          </Typography>
+          <ToggleButtonGroup
+            value={targetApp.annotationCheck}
+            exclusive
+            onChange={handleAlignment}
+            aria-label="text alignment"
+          >
+            <ToggleButton
+              className={classes.annotationToggleBtn}
+              style={{
+                backgroundColor: targetApp.annotationCheck
+                  ? theme.palette.success.main
+                  : theme.palette.disabledBackground,
               }}
-            />
-            <br />
-            <InputField
-              label="appns"
+              value
+              aria-label="left aligned"
+            >
+              <Typography
+                style={{
+                  color: targetApp.annotationCheck
+                    ? theme.palette.common.white
+                    : theme.palette.text.disabled,
+                }}
+                className={classes.text}
+              >
+                {t('createWorkflow.tuneWorkflow.verticalStepper.true')}
+              </Typography>
+            </ToggleButton>
+            <ToggleButton
+              style={{
+                backgroundColor: !targetApp.annotationCheck
+                  ? theme.palette.error.main
+                  : theme.palette.disabledBackground,
+              }}
+              className={classes.annotationToggleBtn}
+              value={false}
+              aria-label="centered"
+            >
+              <Typography
+                style={{
+                  color: !targetApp.annotationCheck
+                    ? theme.palette.common.white
+                    : theme.palette.text.disabled,
+                  textTransform: 'none',
```

Comment:
please use existing classnames or create new, refrain from inline styling

Reply:
For cases like these, I think it's okay to use inline styling since the property (in this case `targetApp`) is directly accessible on the component page, to handle this in styles we'd have to pass this property on the css file then type it there and then do a conditional logic there. 

That is doable but it doesn't really have any added benefits

Reply:
This is preferred when you have a lot of CSS properties interdependent in this prop, like for example the `fill`, `stroke`, `opacity`, etc everything has to change based on `targetApp`, then it bogs down the component JSX.

Reply:
yes but it makes the code base inconsistent, we could have followed this approach at all places then, for a single value to keep inline styling, if we go ahead with this approach we should allow at all places which manipulates a single css property. I personally think its should be the way too, cc: @arkajyotiMukherjee @whitetiger1399 @Jonsy13 @amityt @SarthakJain26 @vanshBhatia-A4k9 @Saranya-jena @oumkale

Reply:
@amityt you may ignore this change request.

Reply:
It always should depend on the use-case...here it seems appropriate.